hail/src/main/scala/is/hail/expr/types/physical/PTuple.scala

```diff
@@ -95,4 +72,4 @@ class CodePTuple(
     assert(pType.nFields == 3)
     (apply[T](0), apply[U](1), apply[V](2))
   }
-}
\ No newline at end of file
+}
```

Comment:
Added newline, not previously present...we may want to enforce trailing newlines if we aren't already (I didn't create the no-newline version of this class, I don't think)